src/midiEditorCommands.cpp

```diff
@@ -223,22 +304,23 @@ bool cancelPendingMidiPreviewNotesOff() {
 }
 
 // A random access iterator for MIDI events.
-// The template parameter is arbitrary and is only used to specialise the
+// The template parameter is used to specialise the
 // template for different event types (note, CC, etc.). We do this instead of
 // subclassing because iterators need to return their exact type, not a base
```

Comment:
nit: Remove the final sentence of this comment about subclassing because now that this iterator returns different types (not always double), subclassing could never work in any way, shape or form.